src/ssl.c

```diff
@@ -15401,6 +15498,17 @@ void wolfSSL_X509_STORE_CTX_set_error(WOLFSSL_X509_STORE_CTX* ctx, int err)
     (void)err;
 }
 
+#ifdef OPENSSL_EXTRA
+void wolfSSL_X509_STORE_CTX_set_time(WOLFSSL_X509_STORE_CTX* ctx,
```

Comment:
Should the time value set here be used for certificate validation during the SSL/TLS handshake? And with the CertManager API (standalone certificate validation)?

If so, we would need to modify wolfSSL and the CertManager to use this time value if set during certificate validation, right?